packages/core/upload/admin/src/components/EditAssetDialog/PreviewBox/AssetPreview.jsx

```diff
@@ -1,32 +1,27 @@
-/* eslint-disable jsx-a11y/media-has-caption */
 import React, { forwardRef } from 'react';
 
+import MuxPlayer from '@mux/mux-player-react';
 import { Flex } from '@strapi/design-system';
 import { File, FilePdf } from '@strapi/icons';
 import PropTypes from 'prop-types';
-import { styled } from 'styled-components';
+import { styled, useTheme } from 'styled-components';
 
 import { AssetType } from '../../../constants';
-import { usePersistentState } from '../../../hooks/usePersistentState';
 
 const CardAsset = styled(Flex)`
   border-radius: ${({ theme }) => theme.borderRadius} ${({ theme }) => theme.borderRadius} 0 0;
   background: linear-gradient(180deg, #ffffff 0%, #f6f6f9 121.48%);
 `;
 
 export const AssetPreview = forwardRef(({ mime, url, name, ...props }, ref) => {
-  const [lang] = usePersistentState('strapi-admin-language', 'en');
+  const theme = useTheme();
 
   if (mime.includes(AssetType.Image)) {
     return <img ref={ref} src={url} alt={name} {...props} />;
   }
 
   if (mime.includes(AssetType.Video)) {
-    return (
-      <video controls src={url} ref={ref} {...props}>
-        <track label={name} default kind="captions" srcLang={lang} src="" />
```

Comment:
@alexandrebodin This didn't actually make sense. Adding language information to metadata doesn't actually do anything.